packages/palette/src/elements/ProgressDots/ProgressDots.story.tsx

```diff
@@ -0,0 +1,37 @@
+import { storiesOf } from "@storybook/react"
+import React, { useEffect } from "react"
+import { useCursor } from "use-cursor"
+import { ProgressDots } from "./ProgressDots"
+
+const Demo = () => {
+  const { index, handleNext } = useCursor({ max: 5 })
+
+  useEffect(() => {
+    const interval = setInterval(handleNext, 1000)
+    return () => {
+      clearInterval(interval)
+    }
+  }, [])
+
+  return (
+    <>
+      <ProgressDots activeIndex={index} amount={5} my={2} />
+    </>
```

Comment:
I don't feel strongly about this so it's not required. Feel free to resolve if you'd like to keep it as is.

```suggestion
    <ProgressDots activeIndex={index} amount={5} my={2} />
```